setup.py

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 from setuptools import setup
+import metric_learn
```

Comment:
Wouldn't this cause trouble for users who clone the fresh repo and run `python setup.py install` ?